src/main/java/de/tum/in/www1/artemis/domain/MathExercise.java

```diff
@@ -0,0 +1,62 @@
+package de.tum.in.www1.artemis.domain;
+
+import static de.tum.in.www1.artemis.domain.enumeration.ExerciseType.MATH;
+
+import javax.persistence.*;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.artemis.domain.enumeration.AssessmentType;
+import de.tum.in.www1.artemis.domain.enumeration.ExerciseType;
+
+/**
+ * A MathExercise.
+ */
+@Entity
+@DiscriminatorValue(value = "MATH")
+@SecondaryTable(name = "math_exercise_details")
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class MathExercise extends Exercise {
+
+    public String getType() {
+        return "math";
+    }
+
+    @Column(name = "example_solution", table = "math_exercise_details")
```

Comment:
This column already exists as part of the `exercise` base table, so you should reuse that one, instead of creating your own.